src/Action/PostCollectionAction.php

```diff
@@ -42,8 +42,9 @@ public function __construct(SerializerInterface $serializer)
      */
     public function __invoke(Request $request)
     {
-        list($resourceClass, $operationName, , $format) = $this->extractAttributes($request);
-        $context = ['resource_class' => $resourceClass, 'collection_operation_name' => $operationName];
+
+        list($resourceClass, $operationName,$itemOperation, $format) = $this->extractAttributes($request);
```

Comment:
We should fix that in another PR then, it's not related to your filter and is a different issue :)